pnpm-lock.yaml

```diff
@@ -70,10 +70,10 @@ importers:
       prettier-plugin-svelte: ^2.6.0
       svelte: ^3.46.3
     devDependencies:
-      '@sveltejs/adapter-auto': 1.0.0-next.34
-      '@sveltejs/kit': 1.0.0-next.306_svelte@3.46.3
+      '@sveltejs/adapter-auto': 1.0.0-next.40
+      '@sveltejs/kit': 1.0.0-next.328_svelte@3.46.3
```

Comment:
is this ok?  if not we need to find a way to update the lockfile format without bumps happening